x-pack/plugins/security/public/components/form_label.tsx

```diff
@@ -46,13 +46,8 @@ export const FormLabel: FunctionComponent<FormLabelProps> = (props) => {
   useEffect(() => report(isEqual), [isEqual]); // eslint-disable-line react-hooks/exhaustive-deps
 
   return (
-    <EuiFlexGroup responsive={false} gutterSize="xs">
-      <EuiFlexItem grow={false}>{props.children}</EuiFlexItem>
-      {!isEqual ? (
-        <EuiFlexItem grow={false}>
-          <EuiIcon type="dot" color="success" />
-        </EuiFlexItem>
-      ) : undefined}
-    </EuiFlexGroup>
+    <>
+      {props.children} {!isEqual ? <EuiIcon type="dot" color="success" /> : undefined}
```

Comment:
Ah, looks like the vertical alignment styling I suggested was fixing it in Chrome, but not Firefox. In that case, let's decrease the icon size to prevent this from happening by using the `size` prop on `EuiIcon`. I think it ends up looking better and less distracting at this smaller size anyway, and it seems to correct the issue in all browsers.

```suggestion
      {props.children} {!isEqual ? <EuiIcon type="dot" color="success" size="s" /> : undefined}
```